charts/scheduler-extender/templates/scheduler-extender.yaml

```diff
@@ -11,6 +11,11 @@ spec:
     metadata:
       labels:
         app: csi-baremetal-se
+        app.kubernetes.io/name: scheduler-extender
```

Comment:
You should use csi-baremetal as name for telegraf discovery.

Reply:
Done